frameworks/elastic/tests/test_tls.py

```diff
@@ -119,11 +118,3 @@ def test_tls_recovery(elastic_service: Dict[str, Any], service_account: Dict[str
         sdk_recovery.check_permanent_recovery(
             package_name, service_name, pod, recovery_timeout_s=60 * 60
         )
-
-        config.check_elasticsearch_index_health(
-            ".security-7",
-            "green",
-            service_name=service_name,
-            http_password=http_password,
-            https=True,
-        )
```

Comment:
Oh I just noticed this... why was it done?

It looks like one of the CI checks failed on this test.